### DISCUSSION TITLE
Artisan test --coverage pcov

### DISCUSSION THREAD

**Discussion by ol1s** [laravel/framework#41856] [Q&A] [answered]
I'd love to get `php artisan test --coverage` working with pcov as opposed to xdebug, but whenever I run the following command:

```
@php -d pcov.enabled=1 artisan test --coverage
```

I get the error:

```
ERROR  Code coverage driver not available. Did you set Xdebug's coverage mode?
```

If I use phpunit direct, it works:

```
@php -d pcov.enabled=1 vendor/bin/phpunit --coverage-text
```

**Comment by robsontenorio** [comment]
It would be really nice :)

**Comment by mominsid** [comment, marked as answer]
The documentation seems to imply that this works with PCOV. "This feature requires [Xdebug](https://xdebug.org/) or [PCOV](https://pecl.php.net/package/pcov)."

https://laravel.com/docs/9.x/testing#reporting-test-coverage

Is this intended to work? If not, that's ok but I think the documentation is misleading.

**Reply by mominsid** [reply]
Just upgraded a project to L10 and PHPUnit 10. It works. Don't know whether and when this issue was fixed, or if it was just isolated to some specific dependency setup.

**Reply by robsontenorio** [reply]
@mominsid do you mind to share your composer.json ?
